docs/URL.md

```diff
@@ -0,0 +1,64 @@
+# Accumulate URLs
+
+A URL has a scheme, user info, authority, path, query, and fragment:
+
+```
+scheme://userinfo@authority/path?query#fragment
+```
+
+The scheme for Accumulate URLs must be `acc`. The `acc` scheme will be added if
+no scheme is specified, but specifying some other scheme will cause the
+transaction to be rejected.
+
+In general, the user info, query, and fragment components of the URL are not
+used.
+
+The authority of a URL may include a port number, but it will be ignored for
+most purposes. The hostname must not be empty. An empty authority or an
+authority that only includes a port number, e.g. `:80`, will be rejected.
+
+Accumulate is case insensitive. Two URLs that only differ in case are considered
+equal. All URL components are lower-cased before they are converted into chain
+IDs.
+
+The identity chain ID is derived from the hostname:
+
+```go
+chain = sha256(lowerCase(url.Hostname))
+```
+
+The resource chain ID is derived from the hostname and the path:
+
+```go
+chain = sha256(lowerCase(url.Hostname + url.Path))
+```
+
+Routing is derived from the identity chain ID:
+
+```go
+routing = uint64(url.IdentityChain()[:8])
+```
+
+Thus, two URLs with the same authority have the same identity chain and routing,
+and two URLs with the same authority and path have the same resource chain,
+regardless of case.
+
+## Token Issuers
+
+The `ACME` token issuer is built into the protocol. In the future, other token
+issuers can be created. Every token issuer has a token URL, such as `acc://ACME`
```

Comment:
Need to specify if we have a token issuance chain or not.  Likely a token issuance chain would make our lives easier.